### PR TITLE
Update language: DIDs enable cryptographic authentication

### PR DESCRIPTION
**Pull request by kimdhamilton**
Update language to indicate this scenario is enabled
"The controller of a DID can cryptographically authenticate themselves (e.g. DID-based website login)."

Addresses #17

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -141,8 +141,8 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
 an authority (e.g. DNS Registrars).
           </li>
           <li>
-The controller of a DID can cryptographically authenticate themselves
-(e.g. DID-based website login) .
+DIDs enable the controller of a DID to cryptographically authenticate themselves
+(e.g. DID-based website login).
```
Comment (msporny) [inline index.html]: We could even put "DID-based website login using a WebAuthn/FIDO token." into the charter as the example. I expect that @nadalin would prefer that we call out ways in which we intend integration to happen between DIDs and WebAuthn.

**Review by msporny — APPROVED** [review]
LGTM, I'd go a bit further w/ the example.

File: index.html (hunk 1 above)
Comment (kimdhamilton) [reply index.html]: good suggestion

**Comment by nadalin** [comment]
@kimdhamilton the term "themselves" is not clear who/whom you are referencing. I also don't know what "authenticate" means, I assume this can  be done via passowords, otp, etc ?

File: index.html (hunk 1 above)
Comment (peacekeeper) [reply index.html]: The RWoT6 DID Auth paper has a [short section](https://github.com/WebOfTrustInfo/rwot6-santabarbara/blob/master/final-documents/did-auth.md#webauthn) on one way of doing DID+WebAuthn integration (there may be several different ways), and IIW26 had a session with a [few slides](https://drive.google.com/drive/folders/1LyYp_SZpqboIPfUa1lo9zKtNv9SIv-5I) on this topic.

**Pull request merged**